apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/web/GetStartedDesign.md

```diff
@@ -1,19 +1,19 @@
 ### Styles
 
-Fabric gives you access to Segoe, Microsoft’s official typeface, along with the color palette, type ramp, icons, and responsive grid for Office 365. [Learn more](#/styles/web)
+Fluent UI gives you access to Segoe, Microsoft’s official typeface, along with the color palette, type ramp, icons, and responsive grid. [Learn more](#/styles/web)
```

Comment:
Ideally we should clarify that the responsive grid part only refers to core, but that may not be as important.